snap/bin/kong-wrapper.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh -e
+
+# need to add luajit/lib folder so that luajit will load properly
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$SNAP/luajit/lib"
```

Comment:
For future shell work, you might want to consider using shellcheck, recommended to me by Alfonso.  I don't think it's worth it to apply to all the scripts in this PR, but we should consider using it for Dehli on all shell scripts.

Reply:
Yeah I've used it before a few times, but didn't explicitly apply it to any of the scripts here